### PR TITLE
Fixes #32 Adds new theme and countdown

### LINKED ISSUE #32: Update the hackathon theme
CAUTION: This is not _theme_ in the sense of css styles but of the subject matter that the hackathon is based upon. Last year was Hotwire and this year is "Supporting the Ruby on Rails Community" and the site should reflect this new theme.

### PR DESCRIPTION
**Pull request by Kentasmic**
Adds this years theme and adds a countdown to the Hackathon start date/time. 
(Note, we will have to update this at 7 PM CST to reflect the end date unless we do this in code.)

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -25,48 +25,37 @@
...
-                <path class="letter" d="m287.75 29.83 1.94-11.1h-29.95l-8.28 47.53h30.49l1.93-11.1h-16.7l1.39-7.95h12.77l1.8-10.25h-12.77l1.22-7.13z"></path>
-                <path class="bolt" d="m211.75 23-11.57 2.69 10.09-25.69-30.2 38.92 14.41-2.84-13.59 26.54 7.86-1.77-7.95 18.95 24.97-26.97-9.21 2.13z"></path>
-              </svg>
-            <% end %>
-          </div>
           <%= link_to "Sign up", new_user_registration_path, class: "inline-flex justify-center rounded-2xl bg-blue-600 p-4 text-base font-semibold text-white hover:bg-blue-500 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70 mt-10 w-full sm:hidden" %>
```
Comment (excid3) [inline app/views/home/index.html.erb:44]: We should comment this out until the database is updated with the years.

Comment (excid3) [reply app/views/home/index.html.erb:44]: It's only visible on mobile btw

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
         <%= link_to "Teams", teams_path, class: "inline-flex justify-center rounded-2xl p-4 text-base font-semibold text-blue-600 hover:bg-blue-200 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
         <%= link_to "Login", new_user_session_path, class: "inline-flex justify-center rounded-2xl p-4 text-base font-semibold text-blue-600 hover:bg-blue-200 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
-        <%= link_to "Sign up", new_user_registration_path, class: "inline-flex justify-center rounded-2xl bg-blue-600 p-4 text-base font-semibold text-white hover:bg-blue-500 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
+        <!-- <%= link_to "Sign up", new_user_registration_path, class: "inline-flex justify-center rounded-2xl bg-blue-600 p-4 text-base font-semibold text-white hover:bg-blue-500 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %> -->
```
Comment (excid3) [inline app/views/home/index.html.erb:20]: Protip: You can use `<%# ` erb comments instead of HTML comments here.

Comment (cjilbert504) [reply app/views/home/index.html.erb:20]: I showed him that last week but guess he forgot and also old habits I suppose lol

**Pull request merged**